grpc-gateway/client/deviceSubscriber.go

```diff
@@ -32,6 +34,8 @@ type Operations interface {
 	UpdateResource(ctx context.Context, event *events.ResourceUpdatePending) error
 	DeleteResource(ctx context.Context, event *events.ResourceDeletePending) error
 	CreateResource(ctx context.Context, event *events.ResourceCreatePending) error
+	// subscriber is not able to reconnect server and must be closed.
```

Comment:
not able to reconnect server?
who is a subscriber? Other CoAP Device or whatever client (grpc, coap, HTTP?)

Reply:
fixed